### PR TITLE
[code42-363] remove-using-statements-from-test-playbook

### PR DESCRIPTION
**Pull request by unparalleled-js**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Maybe an internal issue on your side regarding a failing build for the Code42 Pack

## Description
I heard from @trorabaugh that the build is failing for the Code42 pack because of these using statements in our test playbook `playbook-Code42-Test.yml`.  This PR removes those using statements as well as updates to the latest docker tag (so it is a valid Pack).

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   -  [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by unparalleled-js** [comment]
Would it make sense to change the github user from nathanhusted to mine, since I have been point person on our partner Pack?

File: Packs/Code42/Integrations/Code42/Code42.yml [hunk 1]
```diff
@@ -787,7 +787,7 @@ script:
     - contextPath: Code42.DepartingEmployee.DepartureDate
       description: The departure date for the Departing Employee.
       type: Unknown
-  dockerimage: demisto/py42:1.0.0.11140
+  dockerimage: demisto/py42:1.0.0.12043
```
Comment (orhovy) [inline Packs/Code42/Integrations/Code42/Code42.yml:790]: - Updated Docker image to `1.0.0.12043`.

File: Packs/Code42/ReleaseNotes/2_0_6.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Code42
+- Use latest Docker tag
```
Comment (orhovy) [inline Packs/Code42/ReleaseNotes/2_0_6.md:4]:
```suggestion
- Updated Docker image to `1.0.0.12043`.
```

File: Packs/Code42/Integrations/Code42/Code42.yml (hunk 1 above)
Comment (orhovy) [inline Packs/Code42/Integrations/Code42/Code42.yml:790]: as written in the validation step, there is a new docker image:
Packs/Code42/Integrations/Code42/Code42.yml: [DO106] - The docker image tag is not the latest numeric tag, please update it.
The docker image tag in the yml file is: 1.0.0.12043
The latest docker image tag in docker hub is: 1.0.0.12174
please update to 1.0.0.12174 and update release notes to the newer verstion

Comment (unparalleled-js) [reply Packs/Code42/Integrations/Code42/Code42.yml:790]: Yep, note that this was the latest when I opened the PR and another has come out since

**Review by orhovy — APPROVED** [review]

**Pull request merged**